src/pages/tow.tsx

```diff
@@ -43,7 +43,7 @@ const TowDashPage = ({ calls, endCall, ...rest }: Props) => {
       <Seo title={lang.tow.tow_dashboard} />
 
       <h3>
-        {lang.tow.tow_dashboard} - AOP: {aop}
+        {lang.tow.tow_dashboard} {props.cadInfo?.show_aop === "1" ? `- AOP: ${aop}` : null}
```

Comment:
Thanks for adding this! There's more than just doing this, I'll close this PR and quickly update it myself 😄